src/pages/PageNotFound404.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Link, useLocation } from "react-router-dom";
+
+export default function PageNotFound404() {
+  /************************************
+   * Render
+   ************************************/
+  let location = useLocation();
+
+  return (
+    <div className="container-404">
```

Comment:
This all works for now for basic functionality but it would be nice to have UX take a look. Some styling and copy here could have it go a long way

Reply:
Definitely, I would design something but I'm not on the up & up yet with the current Oneleif brand direction, so just stubbed out for now. I'll post in #uxd to see if anyone else has any ideas!